src/components/error/Error.tsx

```diff
@@ -0,0 +1,38 @@
+import './Error.css'
+import React from 'react'
+import spilledTea from "../../images/Coffee-Burst.svg";
+import { useEffect  } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+interface Props {
+    isRedirected: boolean | any;
+    setIsRedirected: React.Dispatch<React.SetStateAction<boolean | undefined>>
+}
+
+const Error: React.FC<Props> = ({ isRedirected, setIsRedirected }) => {
+    // let location = useLocation()
+    // console.log(location)
+    const navigate = useNavigate()
```

Comment:
According to [router docs](https://reactrouter.com/en/main/hooks/use-navigate), it might be better to use a different method here like redirect.